app_python/docker-compose-server.yml

```diff
@@ -23,3 +23,31 @@ services:
       - ./.docker/config.json:/config.json
     command: ap-api
     <<: *common
+
+  loki:
+    image: grafana/loki:2.6.1
+    container_name: ap-loki
+    command:
+      - -config.file=/etc/loki/local-config.yaml
+      - -config.expand-env=true
+    <<: *common
+
+  promtail:
+    image: grafana/promtail:2.6.1
+    container_name: ap-promtail
+    command:
+      - -config.file=/etc/promtail/config.yaml
+      - -config.expand-env=true
+    environment:
+      - APP_API_LOKI_URL=http://loki:3100/loki/api/v1/push
```

Comment:
interesting ![](https://steamcommunity-a.akamaihd.net/economy/emoticon/:ohh_yeah:)
yeah kinda makes sense to move it here since `http://loki:3100` comes from that file